tests/migrator/xml_imports/test_migrate_proceedings.py

```diff
@@ -72,37 +72,29 @@ def test_migrate_record(datadir, base_app):
                 'Karen Yagdjian: Integral transform approach to time-dependent partial differential equations.'],
             'abstract': 'This book collects lectures given by the plenary speakers at the 10th International ISAAC Congress, held in Macau, China in 2015. The contributions, authored by eminent specialists, present some of the most exciting recent developments in mathematical analysis, probability theory, and related applications. Topics include: partial differential equations in mathematical physics, Fourier analysis, probability and Brownian motion, numerical analysis, and reproducing kernels. The volume also presents a lecture on the visual exploration of complex functions using the domain coloring technique. Thanks to the accessible style used, readers only need a basic command of calculus.',
             '_migration': {
+                **get_helper_dict(record_type="document"),
```

Comment:
about this, I have left it as a raw dict for a reason here, in order to not be able to change internal migration fields so easily as only to change them in the helper dict. Its because then many things depend on this dictionary and I would us to be reminded about that by failing tests